ais-infra-alerting/template.yaml

```diff
@@ -0,0 +1,393 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+
+Description: >-
+  This stack sets up the resources required for AIS alerting
+
+Parameters:
+  Environment:
+    Description: The name of the environment to deploy to
+    Type: String
+    AllowedValues:
+      - "dev"
```

Comment:
Hey readme doesn't mention dev as a deployable environment. I'm just wondering if we want to also deploy this in dev in order to create the SNS topics as the AIS alarm template will try to resolve SNS parameter values in dev

Reply:
Hey, yeah happy with that idea, will add into the README and deploy in dev too :)